Source/Core/Models/Event.cs

```diff
@@ -81,6 +81,7 @@ public static class KnownDataKeys {
             public const string SubmissionMethod = "@submission_method";
             public const string SessionEnd = "sessionend";
             public const string SessionHasError = "haserror";
+            public const string ManualStackingKey = "@manual_stacking_key";
```

Comment:
@ejsmith do you like this name? I kind of like `@override_stacking_hash` or `@advanced_stacking`. Since we have live with this name for a really long time, I'd like some kind of overall consensus? @adamzolotarev thoughts?

I'm wondering what we would display to the user to show how it was stacked? What if a user wanted to stack it on a few different things.. would it be a key value pairs in a commented list or what?